src/pages/cart/SideBar.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from 'react';
+import ItemPrice from './ItemPrice';
+import './SideBar.scss';
+
+const SideBar = props => {
+  const { totalPrice } = props;
+
+  const [deliveryCost, setDeliveryCost] = useState(0);
+
+  useEffect(() => {
+    setDeliveryCost(totalPrice < 30000 ? (totalPrice ? 3000 : 0) : 0);
+  }, [totalPrice]);
```

Comment:
```suggestion

const deliveryCost = totalPrice < 30000 ? (totalPrice ? 3000 : 0) : 0;
```
배송비는 state로 관리하지 않아도 됩니다!

Reply:
수정 완료 했습니다! 감사합니다.